obj_gen.cpp

```diff
@@ -299,33 +299,10 @@ void object_generator::alloc_value_buffer(void)
                 assert(m_random_fd != -1);
             }
 
-            char buf1[64];
```

Comment:
if for some reason we are going to contribute it .. i think we have to keep commit message to the point of what is been fixed :)